apps/web/lib/partnerstack/import-customers.ts

```
@@ -0,0 +1,249 @@
+          });
+        }),
+      );
+    }
+
+    await new Promise((resolve) => setTimeout(resolve, 2000));
+
+    processedBatches++;
+    currentStartingAfter = customers[customers.length - 1].key;
+  }
+
+  delete payload?.startingAfter;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.



(lint/performance/noDelete)